<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioConfigurationDialog</class>
 <widget class="QDialog" name="AudioConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>303</width>
    <height>126</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Audio Interface</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="audioIFComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Sample Rate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sampleRateComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Buffer Size</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="bufferSizeSlider">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>4096</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bufferSizeLabel">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ApplyButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>OkButton</sender>
   <signal>clicked()</signal>
   <receiver>AudioConfigurationDialog</receiver>
   <slot>OkClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>AudioConfigurationDialog</receiver>
   <slot>CancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ApplyButton</sender>
   <signal>clicked()</signal>
   <receiver>AudioConfigurationDialog</receiver>
   <slot>ApplyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>292</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>494</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bufferSizeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AudioConfigurationDialog</receiver>
   <slot>BufferSizeSliderChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>audioIFComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>AudioConfigurationDialog</receiver>
   <slot>audioInterfaceChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>60</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OkClicked()</slot>
  <slot>CancelClicked()</slot>
  <slot>ApplyClicked()</slot>
  <slot>BufferSizeSliderChanged(int)</slot>
  <slot>audioInterfaceChanged(int)</slot>
 </slots>
</ui>
